tests/fixtures/runner.py

```diff
@@ -15,3 +15,18 @@ def runner(tmp_path, tap: Tap, target: Target) -> Generator[Runner, None, None]:
         target=target,
         state_manager=StateManager(base_path=str(tmp_path)),
     )
+
+
+@pytest.fixture
+def runner_with_deselected_stream(
```

Comment:
Zou kijken als je de tap aanpast naar twee streams, dat je in de standaard runner fixture 1 stream deselect. Moet je wel waarschijnlijk wat andere tests links en rechts fixen.